docs/guides/flowing-cred.md

```diff
@@ -1,5 +1,69 @@
+
+
+Edges are directional, indicating the flow of Cred. For instance, when you created your Discourse post node, you get an "inbound" edge from that post, flowing Cred to you. If you referenced another contributor in your post, it would create an “outbound” edge from that post to the referenced contributor, that would flow Cred to them. This makes up the Contribution Graph.
+
+SourceCred converts this graph into a numerical score via a modified [PageRank algorithm](http://ilpubs.stanford.edu:8090/422/1/1999-66.pdf). Every node receives Cred from every node that connects to it. The amount of Cred a node receives is the same as the amount that it sends to other nodes. This means that being connected to a high-Cred node is much more valuable than being connected to a low-Cred node, especially if that high-Cred node isn't connected to many other nodes. In other words, Cred accumulates at important nodes. 
+
+For example, a major contributor is connected to each of their many contributions, and so has a lot of Cred. A spam post, however, may be connected to almost no other nodes, so it will have very little Cred.
+
+----------
+## 💧How Cred Flows
+You can visualize Cred flowing through the graph as water flowing through an ecosystem of ponds and creeks. Cred does not accumulate in one place, but rather flows through each pond, constantly moving. It flows between contributor ponds (Cred Flowing), falls as rain (Cred minting) and evaporates from everyone's ponds over time (Cred evaporation).
+
+### 🏄🏾 Cred Flowing
+Every contributor has their own pond, which represents their Cred score. Your Cred Score is the amount of Cred that is flowing *through* your pond - the amount entering is the same amount that is simultaneously leaving, flowing to other community members. Each of your contributions is interconnected with other community contributions across different platforms. Contributions are even connected through time - today's contributions connect to contributions from yesterday, last week, etc.
+
+### ☔ Cred Minting
+If community members find one of your contributions valuable, they will **Mint**, or in other words, create, Cred, and flow it to you. This Cred  magically appears out of nowhere and flows in your direction. Cred minting is just like rain, new water falling from the sky into your pond. Keep reading to gain a deeper understanding of where Minted Cred comes from.
+
+Example:<br/>
+When someone Mints Cred by reacting to your Discord post with an emoji...
+
+- It creates a new reaction node in the graph
+- It creates an edge from the person reacting (you), to the reaction node
+- It creates an edge pointing from the reaction node to the post receiving the reaction
+  
+
+### 💡 Plug-ins
+
+The types of contributions and plugins tracked by the SourceCred algorithm are likely to increase. Currently, your contributions are tracked across the these active plugins in the following ways:
```

Comment:
```suggestion
The types of contributions and plugins tracked by the SourceCred algorithm are likely to increase. Currently, contributions can be tracked across the these active plugins in the following ways:
```

Rewriting to assume general audience. Unless this page is directed specifically at contributing to SourceCred community, in which case ignore.